<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/home_download.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>Download any type of file</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/youtube.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>Download Video from YouTube</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/playlist.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>Download YouTube Playlist</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/home_settings.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>Themes &amp; Settings</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>File Download</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Paste Download URL here</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Use Button to choose save  location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>170</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="photo.qrc">
        <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Video Download</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>721</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>VIdeo</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>491</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Paste YouTube video URL here</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>280</y>
          <width>551</width>
          <height>16</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>130</y>
          <width>491</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Use Button to browse  save location</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>80</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>200</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>200</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_17">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>350</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_20">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>130</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>310</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Playlist</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>80</y>
          <width>521</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Paste YouTube Playlist URL here</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>370</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>471</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Use Button to browse save location</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>230</y>
          <width>511</width>
          <height>16</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>280</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current Video :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_16">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>130</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>260</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>280</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Total Videos :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>190</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>360</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>190</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Quality</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>260</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <italic>false</italic>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>330</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Theme</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>200</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/youtube.png</normaloff>:/icons/youtube.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/download_file.png</normaloff>:/icons/download_file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
